spec/lib/submission_builder/ty2021/documents/schedule_eic_spec.rb

```diff
@@ -48,6 +48,37 @@
     expect(dependent_nodes[1].at("MonthsChildLivedWithYouCnt").text).to eq "08"
   end
 
+  context "checkboxes 4a and 4b" do
```

Comment:
these tests are actually repetitive with the ones above but i like showing that different cases are being tested with the context blocks instead of all of the fields lumped together? idk